src/app/games/encapsule/encapsule.component.ts

```diff
@@ -171,20 +180,17 @@ export class EncapsuleComponent extends RectangularGameComponent<EncapsuleRules,
     }
 
     public getPieceRadius(piece: EncapsulePiece): number {
-        switch (piece.getSize()) {
-            case Size.BIG:
-                return (this.SPACE_SIZE / 2) - (1 * this.STROKE_WIDTH);
-            case Size.MEDIUM:
-                return (this.SPACE_SIZE / 2) - (2 * this.STROKE_WIDTH) - 3;
-            default:
-                Utils.expectToBe(piece.getSize(), Size.SMALL);
-                return (this.SPACE_SIZE / 2) - (3 * this.STROKE_WIDTH) - 6;
-        }
+        const size: number = piece.getSize();
+        // The aim is that the inter-piece space is half a piece stroke
+        // and the radius is used like this (from center to outside):
+        // empty - stroke - stroke - empty - stroke - stroke
+        // and stick to the square's inner-edge
```

Comment:
tu veux que je te donne la "formulation claire", mais je comprends rien à ce commentaire donc difficile ! Que signifie ce commentaire ? Tu sais expliquer dans d'autres mots pour me faire comprendre ?
Pourquoi tu parle de rayon et tu donne des trucs non liés au rayon ?
Pour le "and stick [...]", j'imagine que tu veux dire "The radius should stick to the square's inner edge"

Reply:
Si l'espace est divisé en zone de taille égale qui ne contiennent qu'un contenu et son cités dans l'ordre, leur contenu est
rien - stroke - stroke - rien - stroke - stroke, aka l'espace entre deux cercles et d'un demi-stroke de rayon, et le rayon du plus petit cercle est de un stroke ! (on ne parle pas de stroke normal de 8 mais de stroke calculé pour utiliser tout l'espace en respectant cette règle)

Reply:
plusieurs fact plus court explicant la logique graphique:
- le stroke d'un cercles est de 2N, les cercles sont espacés de N.
- le rayon du plus grand cercles est tel que son stroke extérieur est inscrit dans un carré de (SPACE_SIZE - HALF_STROKE)

Reply:
Quid de ceci?

> We want a stroke such that:
>   - the stroke of the biggest piece still fits within a space of (SPACE_SIZE - HALF_STROKE)
>   - the spacing between concentric circles is twice as big as their stroke